app/app/local.env

```diff
@@ -6,6 +6,7 @@
 ################################################################################
 
 CACHE_URL=dbcache://my_cache_table
```

Comment:
change the CACHE_URL, above?

Reply:
I was planning on removing it entirely following merging this PR and vetting that the caching isn't messed up.  https://github.com/gitcoinco/web/pull/1969/files#diff-3f9737c05a00f8739afebff6239e18f4L274 makes this field irrelevant from the standpoint of active caching in favor of `REDIS_URL` used here: https://github.com/gitcoinco/web/pull/1969/files#diff-3f9737c05a00f8739afebff6239e18f4R323